floris/simulation/flow_field.py

```diff
@@ -678,22 +669,26 @@ def calculate_wake(self, no_wake=False, points=None, track_n_upstream_wakes=Fals
                     )
                     idx = int(np.where(np.logical_and(yloc, xloc))[0])
 
+                    # placeholder for TI/stability influence on how far
+                    # wakes (and wake added TI) propagate downstream
+                    downstream_influence_length = 15 * turbine.rotor_diameter
+
                     if (
                         coord_ti.x1 > coord.x1
                         and np.abs(coord.x2 - coord_ti.x2) < 2 * turbine.rotor_diameter
+                        and coord_ti.x1 <= downstream_influence_length + coord.x1
```

Comment:
The following could be used to reduce line counts and do the two comparisons at once (pretty minor, so this could be rejected if the change is  opposed for any reason):
```python
                        downstream_influence_length + coord.x1 >= coord_ti.x1 > coord.x1
```